### PR TITLE
Don't multiprocess when testing drizzle

### PR DESCRIPTION
**Pull request by rosteen**
Hopefully this will fix the failing test 🤞

### CODE REVIEW

File: pyproject.toml [hunk 1]
```diff
@@ -107,6 +107,10 @@ filterwarnings = [
     # remove after drizzlepac is updated
     'ignore: `photutils.DAOStarFinder` is a deprecated alias:DeprecationWarning',
     'ignore: Matplotlib is currently using agg, which is a non-GUI backend, so cannot show the figure.:UserWarning',
+    # Ignore warning from astropy.io
+    'ignore: Card is too long, comment will be truncated.',
```
Comment (larrybradley) [inline pyproject.toml:111]:
```suggestion
    'ignore: Card is too long, comment will be truncated.: VerifyWarning',
```

File: pyproject.toml [hunk 2]
```diff
@@ -107,6 +107,10 @@ filterwarnings = [
...
+    # Ignore cast warning in drizzle
+    'ignore: invalid value encountered in cast'
```
Comment (larrybradley) [inline pyproject.toml:113]:
```suggestion
    'ignore: invalid value encountered in cast: RuntimeWarning'
```

File: pyproject.toml (hunk 1 above)
Comment (larrybradley) [reply pyproject.toml:111]: not critical, but you can add the warning class at the end

**Review by larrybradley — APPROVED** [review]
LGTM, thanks!

File: pyproject.toml [hunk 3]
```diff
@@ -108,9 +108,9 @@ filterwarnings = [
     'ignore: `photutils.DAOStarFinder` is a deprecated alias:DeprecationWarning',
     'ignore: Matplotlib is currently using agg, which is a non-GUI backend, so cannot show the figure.:UserWarning',
     # Ignore warning from astropy.io
-    'ignore: Card is too long, comment will be truncated.',
+    'ignore: Card is too long, comment will be truncated.:VerifyWarning',
```
Comment (larrybradley) [inline pyproject.toml:111]: Ah, sorry, `VerifyWarning` is not a built-in warning, so the tests are failing....  remove.

Comment (larrybradley) [inline pyproject.toml:111]:
```suggestion
    'ignore: Card is too long, comment will be truncated.',
```

**Comment by rosteen** [comment]
I'm going to merge this and open a follow-up issue to look into the cast warning.

**Pull request merged**